### ISSUE TITLE
Accessibility Audit: Redundant focus and announcement when user navigates to Senior Housing option on Filter modal

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#951]
**ENVIRONMENT**

macOS 11.6 | Chrome Version 95.0.4638.69 (Official Build) (x86_64) | Voiceover

**ISSUE**

Turn on Screen Reader, and go to the [Listings](https://detroit-public-dev.netlify.app/listings) page.  Open the Filter modal and  navigate to the 'Senior Housing (62+)' option. Observe the behavior and announcement. As you can see from the attached video, controls and associated text have redundant focus when user navigates to this option. Voiceover announces 'Senior Housing (62+)' when focus is on the text element and announces it again when the focus is on the popup button. Ideally the control and associated text should have a single focus and screen reader should announce it once.

https://user-images.githubusercontent.com/394908/148572221-ed1b06f7-9c09-4633-8917-c91133728e68.mov
